inst/resources/css/thesis-page.css

```diff
@@ -0,0 +1,433 @@
+/* This template was modified by Brent Thorne from the 'default-page.css, and default.css' file please see the wonderful work done by Yihui Xhe and Romain Lesaur on that package at https://github.com/rstudio/pagedown */
```

Comment:
I can go through but this may take some time. I will let you know when I have time to do it, thanks.

Reply:
This is a minor issue (this could help to maintain this template in the long term). 
Since you consider this can take some time, do not do this: the benefit is not high enough.